integration-core/src/main/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeImpl.java

```diff
@@ -102,6 +95,12 @@
       List<String> streams, Message message) throws RemoteApiException {
     List<Message> result = new ArrayList<>();
 
+    if (streams.isEmpty()) {
```

Comment:
In this case we should return a 404 not found, to be compatible with the case where we return 404 when the bot is removed from the room.